src/serialization.jl

```diff
@@ -27,7 +88,11 @@ function restoreoptsum!(
     if length(setdiff(allowed_missing, keys(dict))) > 1 # 1 because :lowerbd
         @warn "optsum was saved with an older version of MixedModels.jl: consider resaving."
     end
+
     if any(ops.lowerbd .> dict.initial) || any(ops.lowerbd .> dict.final)
+        @debug "" ops.lowerbd
+        @debug "" dict.initial
+        @debug "" dict.final
```

Comment:
I don't know whether you separated them on purpose but FYI you can combine them into one `@debug`:
```suggestion
        @debug "" ops.lowerbd dict.initial dict.final
```